source/_integrations/cast.markdown

```diff
@@ -233,6 +233,83 @@ To cast media directly from a configured Plex server, set the fields [as documen
       media_content_id: 'plex://{"library_name": "Movies", "title": "Groundhog Day"}'
 ```
 
+### [BBC Sounds](https://www.bbc.co.uk/sounds)
+This app doesn't retrieve its own metadata, so if you want the cast interface or media player card to show titles and/or images you will have to provide the data yourself. See the examples below.
+
+#### Media parameters
+Mandatory:
+- `app_name`: `bbcsounds`
+- `media_id`: Item ID
+
+Optional:
+- `is_live`: Item is a livestream
+
+#### Example:
+Example values to cast the item at <https://www.bbc.co.uk/sounds/play/live:bbc_radio_one>
+```yaml
+  alias: "Cast BBC Sounds to My Chromecast"
+  sequence:
+    - target:
+        entity_id: media_player.my_chromecast
+      data:
+        media_content_type: cast
+        media_content_id: '
+          {
+            "app_name": "bbcsounds",
+            "media_id": "bbc_radio_one",
+            "is_live": true
+          }'
+        extra: 
+          metadata: 
+            metadataType: 0
+            title: "Radio 1"
+            images:
+              - url: "https://sounds.files.bbci.co.uk/2.3.0/networks/bbc_radio_one/background_1280x720.png"
+      service: media_player.play_media
+```
+
+### [BBC iPlayer](https://www.bbc.co.uk/iplayer)
+This app doesn't retrieve its own metadata, so if you want the cast interface or media player card to show titles and/or images you will have to provide the data yourself. See the examples below.
+
+Note: Media ID is NOT the 8 digit alpha-numeric in the URL, it can be found by right clicking the playing video. E.g. <https://www.bbc.co.uk/iplayer/episode/b09w7fd9/bitz-bob-series-1-1-castle-makeover> shows:
```

Comment:
```suggestion
### [BBC iPlayer](https://www.bbc.co.uk/iplayer)

This app doesn't retrieve its own metadata, so if you want the cast interface or media player card to show titles and/or images you will have to provide the data yourself. See the examples below.

Note: Media ID is NOT the 8 digit alpha-numeric in the URL, it can be found by right-clicking the playing video. E.g., <https://www.bbc.co.uk/iplayer/episode/b09w7fd9/bitz-bob-series-1-1-castle-makeover> shows:
```